### ISSUE TITLE
[18.0] stock: Lots / Serial Numbers

### ISSUE THREAD

**Issue by rousseldenis** [odoo/odoo#190874]
### Odoo Version

- [X] 16.0
- [X] 17.0
- [X] 18.0
- [ ] Other (specify)

### Steps to Reproduce

@amoyaux @Elftortom

When managing drugs (and GS1 barcodes), you can have a product instance with a GTIN reference (EAN), a lot, a serial number and an expiration date.

Currently, Odoo does not allow to have a `stock.lot`instance with a different value for lot and for serial number.

Is it scheduled to support that?

Many thanks

**Comment by gdgellatly** [comment]
I have a similar issue, I have not looked in detail beyond some initial trial after some advice from Odoo, have you considered packages as the lot?

**Comment by cvinh** [comment]
Same for us. A workaround would be to search lot or serial @etobella

**Comment by vehi-invitu** [comment]
> Same for us. A workaround would be to use serial which is supposed to be more unique than lot... [@etobella](https://github.com/etobella)

IMHO, I believe that we should use the tracking field (if the product is tracked by lots, the lot number is used; if it’s tracked by serial number, then the serial number is used instead).

**Comment by cvinh** [comment]
Yes in our use case we have to choose by product... but having both doesn't seem to be in odoo's roadmap
